src/definitions.ts

```diff
@@ -5,5 +5,17 @@ declare module "@capacitor/core" {
 }
 
 export interface SignInWithApplePlugin {
-  Authorize(): Promise<{response: any}>;
+  Init(options: InitOptions): Promise<void>;
+
+  Authorize(): Promise<{ response: any }>;
+
+  SignIn(): Promise<void>;
```

Comment:
Maybe remove this - you don't need it (see other comment)

Reply:
Updated with db36e4f4f5497465dc393012e993a4e86fce8b50